lib/mri/mkmf.rb

```diff
@@ -10,6 +10,10 @@
 # For TruffleRuby
 require 'pathname'
 
+unless ENV['SULONG_HOME']
```

Comment:
Can you put `# For TruffleRuby` immediately above this? It makes it more clear when we merge and have to reapply the changes.